src/routes/ModListPage.js

```diff
@@ -1,13 +1,125 @@
 import React from 'react';
+import InfiniteScroll from 'react-infinite-scroll-component';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faDownload } from '@fortawesome/free-solid-svg-icons';
+import { faUserCircle, faHeart, faFolderOpen, faArrowAltCircleDown, faWindowRestore } from '@fortawesome/free-regular-svg-icons'
+
+// Components
+import { SearchSection, SearchContentItem, SearchContentBtn } from '../components/SearchSection';
 
 // Stylesheets
-import '../stylesheets/mods.css';
+import '../stylesheets/modList.css';
 
 const ModListPage = () => {
+  const [items, setItems] = React.useState(Array.from({ length: 20 }));
+  const [hasMore, setHasMore] = React.useState(true);
+
+  const fetchMoreData = () => {
+    if (items.length >= 300) {
+      setHasMore(false);
+      return;
+    }
+
+    setTimeout(() => {
+      setItems(items.concat(Array.from({ length: 20 })));
+    }, 500);
+  };
+
   return (
     <>
-      <div className="mods-list-main">
-        attachments
+      <div className="mods-main">
+        <SearchSection
+          placeholder='Search to Mod Name'
```

Comment:
```suggestion
          placeholder='Search Mod'
```
better than 'search to mod name'

Reply:
fixed